core/src/modules/ops/export.ts

```diff
@@ -99,7 +100,9 @@ export namespace ExportOps {
 
     _exports = await Export.findAll({
       where: {
-        startedAt: null,
+        startedAt: {
+          [Op.or]: [null, { [Op.lt]: new Date().getTime() - delayMs }],
```

Comment:
WIth this `or` it's notable that the code below on 115 isn't updated
```
        where: {
          id: { [Op.in]: _exports.map((e) => e.id) },
          startedAt: null,
        },
```
so in the "stale" case, we aren't really updating the `startedAt` to now.

Reply:
I'm wondering if it might be cleaner for a "Sweeper" job to set their startedAt to `null` instead of the `or` here.

In the ideal case, it would also make sure it wasn't still in the queue, I suppose. Though maybe a queueLock on that is better?